packages/bson-bench/test/unit/task.test.ts

```diff
@@ -13,7 +13,7 @@ describe('Task', function () {
   const BSON_PATH = process.env.BSON_PATH;
   const BSON_EXT_PATH = process.env.BSON_EXT_PATH;
   const versions = [
-    'bson@6.0.0',
+    'bson@6.2.0',
```

Comment:
Changed this as installing 6.0.0 works when the entire test suite is not run together. There is something to look into here as far as the interaction between subsequent runs of `Package.install`, but this does not seem to affect actual usage of the library.